util/helpers.py

```diff
@@ -7,3 +7,19 @@ def secs_to_hours(secs):
     hours, remainder = divmod(secs, 3600)
     minutes, seconds = divmod(remainder, 60)
     return '%d:%02d:%02d' % (hours, minutes, seconds)
+
+def check_ctcdecoder_version():
+    import sys
+    import os
+    import semver
+    from ds_ctcdecoder import ctc_beam_search_decoder, Scorer, __version__ as decoder_version
```

Comment:
Why are you importing all the other symbols too rather than just `__version__`?

Also, since the reason we're doing this is to reduce confusion from users, I would guard this with a try-except and print the compatibility message if `ds_ctcdecoder` does not have `__version__` in it.

Reply:
> Why are you importing all the other symbols too rather than just `__version__`?

Copy/paste error, sorry



> Also, since the reason we're doing this is to reduce confusion from users, I would guard this with a try-except and print the compatibility message if `ds_ctcdecoder` does not have `__version__` in it.

That would be a nice addition.